<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirBattle</class>
 <widget class="QMainWindow" name="AirBattle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>946</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AirBattle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="times">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QLabel" name="timed">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_name">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>P2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="boss_name">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>P1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timem">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="menu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="value1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="value2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>2Tech</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
